ovos_plugin_manager/language.py

```diff
@@ -114,8 +118,6 @@ class OVOSLangDetectionFactory:
         "lingua_podre": "lingua_podre_plug"
     }
 
-    # TODO - get_class method
```

Comment:
These methods could be implemented, but I'm not sure its necessary?

Reply:
it might be useful for things that want to load plugin classes without loading the object itself

some Factory objects implement it some don,t its just a matter of consistency